### ISSUE TITLE
🤡 badge for climate change deniers (just like you highlight government sponsored propaganda) 

### ISSUE THREAD

**Issue by marsrobertson** [twitter/the-algorithm#1863]
### **Is your feature request related to a problem? Please describe.**

Yes.
Climate change is a **problem**.
Climate change denial is a **problem**.
Climate change inaction is a **problem**.
Inaction is caused by denial.

### **Describe the solution you'd like**

Renewable energy. Regen agriculture. Reduction in emissions. End the war. The list can go on and on, check my frens at https://planetarycouncil.org

### **Describe alternatives you've considered**

Blowing up the Parliament, just like Guy Fawkes, such a legend: https://en.wikipedia.org/wiki/Guy_Fawkes_Night

It would get some press, get some attention, also terrorism charges to spend some with Julian Assange _(more than 4 years without trial, judicial kidnapping)_

### **Additional context**
Fake news. Click baits. Time on site. Engagement metrics. Attention economy. Outrage farming. Not good for mental health.

I'm putting 1 BTC on the line and want to debate a climate change denier with 99-1 advantage.

`PSEUDO ALGORITHM`
* can be hacked overnight
* does not require excessive resources
* overtime can be fine tuned to rely more on AI

1. To begin with, only accounts with 100+ followers, 100+ tweets, that tweeted in the past month
2. Generate a list of 1000 climate related words, comma separated (that's the exact prompt)
3. No rocket science (Elon is doing it), simple `for` loop and `if else` statement with a counter. 
4. Total number of tweets is low trillions, shouldn't take long, even written in JavaScript on my Apple Watch
5. Climate words / Total words. Receive a number. Sort by number. Now we know who tweets about the climate.
6. Only use tweets of the user. Do not count replies for now. Loads of trolls.
7. Ask the AI. **"It this tweet OK with scientific consensus, or is it leaning towards climate change denial"**?
8. 3 strikes rule 🤡
9. Then run the algo for all the followers
10. Mars spends some time in HMP Belmarsh with Assange 

_(you need to do it quickly, otherwise the incentive to voluntarily go to prison is no longer attractive)_

FYI. I'm putting 1 BTC on the line and offering 99-1 handicap: https://twitter.com/marsxrobertson/status/1673724721601314816

PS. Russia is the only country that benefits from climate change. Melting Arctic opens Northwest Passage. If you do not act on climate you are Putin supporter 🤡

**Comment by marsrobertson** [comment]
The reason why I suggested 🤡 rather than 💩 is because some people genuine like poo emoji.

We can figure out any other label to climate change deniers, does not have to be emoji,

**Comment by Naim2000** [comment]
Sorry im not reading allat

**Comment by marsrobertson** [comment]
### Trending hashtag appeared in my sphere of consciousness

<img width="555" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/2d63d241-5ba6-407d-a09b-f179e127b035">

### Two top tweets (trivially debunkable)

<img width="566" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/128667a3-fe9e-492e-84ae-b8fc5aae5283">

https://twitter.com/namemeifucan/status/1674491357782503437

<img width="552" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/cf163372-a317-439d-aea7-dfab2a8b3746">

https://twitter.com/theeZR0/status/1674527088789864466

### Tried responding and inviting to 1 BTC 99-1 handicap debate

<img width="561" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/43f8a4fc-dad6-480a-9d1c-36dabf2c4d67">

### Is there already some mechanism in place?

For example, if a tweet receives multiple flags it is automatically not-replyable, not-retweetable?

**Comment by xTrayambak** [comment]
Not reading allat 😒

**Comment by marsrobertson** [comment]
Thanks for educating me, I didn't about this meme: https://knowyourmeme.com/memes/i-aint-reading-all-that

![image](https://github.com/twitter/the-algorithm/assets/44369284/c146da7e-2813-4e92-bd0a-af0b25010e9c)

# TLDR

![image](https://github.com/twitter/the-algorithm/assets/44369284/7ee2141a-757b-4313-8b6f-8be11d07079c)

### 🤡 for climate change deniers

**Comment by xTrayambak** [comment]
Climate change is real, hardly anyone here denies it. Please stop.

**Comment by marsrobertson** [comment]
@xTrayambak 

> hardly anyone here denies it

![image](https://github.com/twitter/the-algorithm/assets/44369284/3738b857-5ee1-4494-a436-c26b47244b87)

My estimate (not scientific method) is that climate change deniers are 0.1% to 1% of the global population but they are getting disproportionally too much airtime.

Again, my estimate (not scientific method) is that sometimes we simplify:
* chances of yes: 50%
* chances of no: 50%
* principle of fairness
* principle of "innocent until proven guilty"

A determined climate change denier can cause loads of harm, confusion, inaction... I'm determined climate change activist and I was frustrated that they were quitting on me. No honest troll would quit a heated debate with a guy like me, it didn't make sense. Putin, Russia, Arctic, Northwest Passage hypothesis came to mind.

# Have you heard of Metacrisis?

![image](https://github.com/twitter/the-algorithm/assets/44369284/672b6658-a64a-40b0-86db-5c2c0d2b4674)

<img width="1405" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/e3857662-5995-444c-86e4-deca59e6a835">

Source: https://fortune.com/2023/02/15/trust-in-media-low-misinform-mislead-biased-republicans-democrats-poll-gallup/

Lost trust in media - lost trust in government. Who do you trust? Twitter is fast to operate with Elon. And the pseudocode algorithm is relatively easy to implement, get started, improve over time... Cambridge Analytica tech is "genie out of the bottle" and maybe there is a way to repurpose it towards something good?

Of course cannot ban climate change denial - free speech - Streisand Effect - that's why 🤡 emoji to be taken seriously and explaining: _"Russia the only country that benefits from melting Arctic, opening Nortwest Passage, fight Putin by fighting climate change"_

**Comment by xTrayambak** [comment]
not reading allat 😄👍

**Comment by xTrayambak** [comment]
tldr: blah blah blah I am very scientific bleh bleh blah blah bleh blah bleh Putin bad (he technically is) blah blah blah free speech blah blah bleh I am very smart give me validation and internet points blah blah bleh

**Comment by marsrobertson** [comment]
![image](https://github.com/twitter/the-algorithm/assets/44369284/03b5cf3d-a269-46f9-a90b-5863833f51cc)

Flat Earth is not dangerous.

Climate change denial / confusion / distraction / inaction is.

**Comment by xTrayambak** [comment]
My guy. I don't believe in God and neither do I believe in the flat earth theory (it doesn't even exist in the religion I've been assigned on paper).

**Comment by CamoCatX** [comment]
Guys, you're both wrong.

![image](https://github.com/twitter/the-algorithm/assets/69330490/3a01f3b3-4560-4083-ae58-e0c08d6c2bb7)


https://www.reddit.com/r/cubeearthsociety/

**Comment by CamoCatX** [comment]
Even Bing search doesn't deny it:
![image](https://github.com/twitter/the-algorithm/assets/69330490/d04584c5-2e48-4b19-ae87-be3ff2cef086)

**Comment by marsrobertson** [comment]
# Looks like a triangle to me

![image](https://github.com/twitter/the-algorithm/assets/44369284/4b7445cc-345d-4b0d-9229-8a9d4ae86288)

**Comment by jackcooperusesvim** [comment]
Bruh. lol. yall are delusional. AI is gonna take over the world in like 3 days.

**Comment by marsrobertson** [comment]
# YES BUT

> We invite you to join us now on the next phase in our journey to realizing a fully decentralized AGI platform, governed in an effective, stable, fair and transparent manner by the widest group possible. In this way, a beneficial Singularity has a chance to change the world.

— [Announcing the Supervisory Council Elections: The Next Step on SingularityNET’s Roadmap to Decentralized Governance](https://blog.singularitynet.io/announcing-the-supervisory-council-elections-the-next-step-on-singularitynets-roadmap-to-9f4d8dc18c6d)

![Screenshot-2023-10-04-at-21 07 15](https://github.com/twitter/the-algorithm/assets/44369284/a07fb3d5-5167-4aca-99d3-762333da45e9)

3 mnutes video: https://youtu.be/fWV6UuFLLso

<img width="741" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/dbfbc757-4f73-48a9-8ae5-b3505d17c2e0">

Applied here as well: https://www.gov.uk/government/news/initial-100-million-for-expert-taskforce-to-help-uk-build-and-adopt-next-generation-of-safe-ai

![image](https://github.com/twitter/the-algorithm/assets/44369284/e2e4c892-cc20-4b1e-a361-1d1f6d40c1ce)

**Comment by xTrayambak** [comment]
Please tell me this is satire.

**Comment by marsrobertson** [comment]
>  Please tell me this is satire.

Genuine satire.

<img width="820" alt="image" src="https://github.com/twitter/the-algorithm/assets/44369284/5679c491-5b40-481d-8cb4-4fd1c7242930">

### https://www.youtube.com/watch?v=kLzPuNL0BKU

> Alright, how do you tell the difference between genuine satire and non genuine satire?
> That's a matter for the courts.

**Comment by jackcooperusesvim** [comment]
marsrobertson really thinks hes the stuff huh.

**Comment by marsrobertson** [comment]
> marsrobertson really thinks hes the stuff huh. — @jackcooperusesvim 

I do not know what I was thinking / really thinking.

You prompted me to think about it.

Quantum mechanics - https://en.wikipedia.org/wiki/Observer_effect_(physics) - observer affecting the observation.

**Comment by marsrobertson** [comment]
Actually... Maybe this?

![IMG_4406](https://github.com/twitter/the-algorithm/assets/44369284/ff2e9ffe-3ad5-4350-9e89-5dcc45e20bb5)

https://en.wikipedia.org/wiki/Occam%27s_razor - the simplest answer is often correct

**Comment by Marioood** [comment]
fucking hell can we just publically execute all twitter users at this point?

**Comment by jackcooperusesvim** [comment]
What kinda fit should I wear to my public execution?

**Comment by marsrobertson** [comment]
Balanced approach.

Someone tweeted: https://x.com/packyM/status/1799548979656433769

![earth flat 2](https://github.com/twitter/the-algorithm/assets/44369284/3c3d0161-76ae-43c5-ae05-99ed6dd8faf3)

**Comment by marsrobertson** [comment]
Tweet: https://x.com/misterduncan/status/1820168563350450324
<img width="630" alt="image" src="https://github.com/user-attachments/assets/2cff1eb0-20d8-4311-9faf-619e281f71a3">

From the news:
<img width="692" alt="image" src="https://github.com/user-attachments/assets/bb4a75b7-5f2e-47e3-a099-886860d6546e">

But the climate deniers 🤡 will always find an explanation:
* natural cycles
* magma eruption
* solar flares
* distance from the sun

Science says It is "unequivocal" but the word is too fancy and not everyone understands it:

<img width="708" alt="image" src="https://github.com/user-attachments/assets/6c37d614-d886-43b7-9ef8-5c00c3991a8a">

On top of that IPCC are scientics who are paid by the universities, governments, institutions who can benefit from A B C X Y Z.

That's why I like photos like this, they tell a story, and with the traffic they also invite trolls...

## Radical 💡💡💡

Climate change denial police bot 🤖👮🚨 Debating at scale.

I enjoy debating and emotions coming out of it, but it can be seen as an unhealthy addiction. Debating at scale that would be something remarkable.

**Comment by marsrobertson** [comment]
One of the most outspoken arguments.

It is clearly a man-made catastrophe, so what do you do with it now?

![IMG_9329](https://github.com/user-attachments/assets/d1a06d9c-2429-42a3-9dd5-502b3a1ae08a)

**Comment by marsrobertson** [comment]
![donut gif4](https://github.com/user-attachments/assets/140f2140-091d-4dae-9fe0-a8b4a1f7df37)

![moebious gif2](https://github.com/user-attachments/assets/8a703367-d327-4fd7-a8ca-32b2f07447ae)

![image](https://github.com/user-attachments/assets/13d0441d-9395-4a2b-a762-64e8d1145d2c)

Meanwhile i'm physically located in a place that captures water and grows its own food.
